Puppetfile

```diff
@@ -11,7 +11,7 @@ mod 'katello/candlepin',               '>= 12.0.0 < 12.1.0'
 mod 'theforeman/pulpcore',             '>= 5.0.0 < 5.1.0'
 mod 'katello/qpid',                    '>= 9.0.0 < 9.1.0'
 mod 'theforeman/foreman',              '>= 18.2.0 < 18.3.0'
-mod 'theforeman/foreman_proxy',        '>= 19.0.0 < 19.1.0'
+mod 'theforeman/foreman_proxy',        '>= 19.0.1 < 19.1.0'
```

Comment:
This is not needed since `>= 19.0.0` also matches `19.0.1`. Just the entry in the lockfile is sufficient.